addons/l10n_ar/i18n/l10n_ar.pot

```diff
@@ -1192,6 +1192,12 @@ msgstr ""
 msgid "FORM 1116 RT"
 msgstr ""
 
+#. odoo-python
```

Comment:
I have no idea but this is different than the entries above and below (if a rebase does not solve it)